docs/release_notes/v0.12.0-alpha.md

```diff
@@ -0,0 +1,74 @@
+# Spice v0.12-alpha (Apr 29, 2024)
+
+The v0.12-alpha release introduces Clickhouse and Apache Spark data connectors, adds support for limiting refresh periods for temporal datasets, and includes upgraded Spice Client SDKs compatible with Spice OSS.
```

Comment:
This also needs a breaking change notice

Reply:
I added this below.